std/http/file_server.ts

```diff
@@ -107,7 +107,7 @@ async function serveFile(
   const [file, fileInfo] = await Promise.all([open(filePath), stat(filePath)]);
   const headers = new Headers();
   headers.set("content-length", fileInfo.len.toString());
-  headers.set("content-type", "text/plain; charset=utf-8");
+  headers.set("content-type", "text/html; charset=utf-8");
```

Comment:
In fact, we should set the `content-type` according to extname, but this will cause inconvenience to browse `.ts` files. In previous versions, some files were downloaded directly instead of displaying the content on the browser, so this was temporarily set to `plain`.